src/common/commands_parser.rs

```diff
@@ -392,6 +396,16 @@ fn debug_mode(args: &ArgMatches) -> Option<bool> {
     }
 }
 
+/// Parse the enclave name from the command-line arguments.
+fn parse_enclave_name(args: &ArgMatches) -> NitroCliResult<Option<String>> {
+    let enclave_name = if let Some(enclave_name) = args.value_of("enclave-name") {
+        Some(enclave_name.to_string())
+    } else {
+        None
+    };
+    Ok(enclave_name)
```

Comment:
```suggestion
    Ok(args.value_of("eif-path").map(|e| e.to_string()))
```

Reply:
will change with "enclave-name" instead of "eif-path"

Reply:
yeah, sorry, i meant "enclave-name"